package.json

```diff
@@ -75,7 +75,7 @@
     "@typescript-eslint/eslint-plugin": "^5.48.0",
     "@typescript-eslint/parser": "^5.48.0",
     "@vitejs/plugin-react": "^4.2.1",
-    "@vitest/coverage-istanbul": "^1.1.1",
+    "@vitest/coverage-v8": "^1.3.1",
```

Comment:
Istanbul plugin had some issues instrumenting our code. v8 works better and also it is the default in vitest now